src/main/java/duke/util/Event.java

```diff
@@ -0,0 +1,48 @@
+package duke.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event
+ *
+ * <p>CS2103T AY19/20 Semester 2
+ * Individual project
+ * Duke project
+ *
+ * 29 Jan 2020
+ *
+ * @author Jel
+ */
+public class Event extends Task {
+    private LocalDate at;
+
+    /**
+     * Constructs an Event instance.
+     * @param description The description or details of the Event.
+     * @param at The date where the Event will happen.
+     */
+    public Event(String description, LocalDate at) {
```

Comment:
Nice! Good constructor. Can be improved by accepting non-localdate time as valid events too, like 'buy dian hao coffee /by tomorrow'.